### ISSUE TITLE
scrollToBottom() max calls

### ISSUE THREAD

**Issue by smakinson** [tangbc/vue-virtual-scroll-list#196]
**Is your feature request related to a problem?**

Before I noticed the example was reversing the flex-direction on item render, the scrollToBottom would get into an infinite loop from the setTimeout.

**Describe the feature request**

I think it would be nice if that method was adjusted to prevent the possibility of infinite calls. Maybe if it's expected to find the bottom within a couple tries they can just be done manually in the method instead of calling itself?

**Comment by tangbc** [comment]
How to repreduce the infinite call scrollToBottom?

```javascript
setTimeout(() => {
  if (this.getOffset() + this.getClientSize() < this.getScrollSize()) {
    this.scrollToBottom()
  }
}, 3)
```
There is a condition make sure in bottom.

**Comment by smakinson** [comment]
I had been looking at this example: https://github.com/tangbc/vue-virtual-scroll-list/blob/master/example/src/views/chat-room/Main.vue

and had the `<virtual-list/>` css set to `flex-direction: column-reverse;`

and had yet to pay attention to what was going on around `:class="{ overflow: overflow }"`

Because of that I was seeing `scrollToBottom()` actually making it scroll to the top and when I went into my node modules and stuck a console log in the `scrollToBottom` method temporarily I found it repeatedly logging. I was also unable to drag the scroll bar down because of the infinite calling.

It was my fault for not fully setting up the CSS, but I was thinking perhaps the method should have a max number of times it could end up doing the `setTimeout` to help avoid locking up the browser when digging into an example.
